### PR TITLE
Add HaTS and Opt-In Feedback links

### PR DESCRIPTION
**Pull request by tejal29**
Relates to _in case of new feature, this should point to issue/(s) which describes the feature_

**Description**

This PR add the Skaffold Survey links to our documentations
- Updates to skaffold.dev
  - Add a link to survey to [What's Next in Getting Started docs](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/docs/content/en/docs/getting-started/_index.md#whats-next) 
  - Add a link to survey in [Tutorials Page] (https://github.com/tejal29/skaffold/blob/add_hats_survey_links/docs/content/en/docs/tutorials/_index.md)
 - Add a new [Skaffold Feedback](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/docs/content/en/docs/references/feedback/_index.md) Page
    - link to HaTS survey
    - link to Skaffold Opt-in Feedback Survey
    - mentions a link to Skaffold Opt-out form.
-  Updates to Readme
    -  Add a link to the survey in [Install Intructions](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/README.md#install)
   -  Add a link to newly created skaffold Feedback Resource in the [Community Section](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/README.md#community)

**User facing changes**

Skaffold docs Changes listed above.

**Before**
- Readme on master
  - [Community](https://github.com/GoogleContainerTools/skaffold/blob/master/README.md#community)
  - [Install](https://github.com/GoogleContainerTools/skaffold/blob/master/README.md#install)

-  docs on master
   -  [Getting Started](https://skaffold.dev/docs/getting-started/#what-s-next)
   -  [Tutorials](https://skaffold.dev/docs/tutorials/)
   -  Feedback page - does not exits.

**After**
- Readme on my branch
  - [Community](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/README.md#community)
  - [Install](https://github.com/tejal29/skaffold/blob/add_hats_survey_links/README.md#install)

- docs on PR
   -  [Getting Started](http://34.94.71.232:1313/docs/getting-started/#what-s-next)
   -  [Tutorials](http://34.94.71.232:1313/docs/tutorials/)
   -  [Feedback page](http://34.94.71.232:1313/docs/references/feedback/)

**Next PRs.**
n/a

**Submitter Checklist**

These are the criteria that every PR should meet, please check them off as you
review them:

- [ n/a] Includes [unit tests](../DEVELOPMENT.md#creating-a-pr)
- [ X] Mentions any output changes.
- [ X] Adds documentation as needed: user docs, YAML reference, CLI reference.
- [ n/a] Adds integration tests if needed.

_See [the contribution guide](../CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a example to the `examples` dir](https://github.com/GoogleContainerTools/skaffold/tree/master/examples), please copy them to [`integration/examples`](https://github.com/GoogleContainerTools/skaffold/tree/master/integration/examples)
- Every new example added in [`integration/examples` dir](https://github.com/GoogleContainerTools/skaffold/tree/master/integration/examples), should be tested in [integration test](https://github.com/GoogleContainerTools/skaffold/tree/master/integration)

**Reviewer Notes**

- [x] The code flow looks good. 
- [x] Unit test added.
- [ ] User facing changes look good.

### CODE REVIEW

File: docs/config.toml [hunk 1]
```diff
@@ -31,6 +31,9 @@ pygmentsStyle = "tango"
  # First one is picked as the Twitter card image if not set on page.
  #images = ["images/project-illustration.png"]
 
+# Enable Emojis
+enableEmoji = true
```
Comment (priyawadhwa) [inline docs/config.toml]: 😄

File: docs/content/en/docs/references/feedback/_index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Your feedback is very important to us to make Skaffold better!
+Sending us feedback is easy.
+
+## Quick HaTS Survey
+Take our [quick 5-question survey](https://forms.gle/BMTbGQXLWSdn7vEs6) to tell us how satisfied you are with
+Skaffold and and what improvements we should make.
+
+## Skaffold Feedback Sessions
+The Skaffold team actively incorporates user feedback as the product evolves.
+We are looking to chat with developers like you to learn about how you use Skaffold and
```
Comment (priyawadhwa) [inline docs/content/en/docs/references/feedback/_index.md]: maybe rephrase to something like:

We are looking to chat with developers like you to learn how you use Skaffold and how we can improve it to better suit your needs.

File: docs/content/en/docs/references/feedback/_index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+If you’re interested, please provide your contact information below by filling in
+[Skaffold feedback form](https://forms.gle/J5h567ncypY2ziq49).
+
+We'll reach out to schedule a one-hour hangouts session at a time convenient for you. Providing your information does not guarantee that we will contact you. 
+
+
+No preparation is needed on your part.
+Plus, you'll get the chance to help build something you'll love and a token of our appreciation once participation  is complete!
+
+If at any time you wish to opt out of any communications from Skaffold team, please fill our [opt-out form](https://forms.gle/cnfCXotenyUtF92w6)
```
Comment (priyawadhwa) [inline docs/content/en/docs/references/feedback/_index.md]: nit: period at the end

File: docs/content/en/docs/references/feedback/_index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Skaffold Feedback"
+linkTitle: "Skaffold Feedback"
+weight: 300
+---
+
+Your feedback is very important to us to make Skaffold better!
+Sending us feedback is easy.
+
+## Quick HaTS Survey
```
Comment (balopat) [inline docs/content/en/docs/references/feedback/_index.md]:
```suggestion
## Quick survey
```

Comment (balopat) [reply docs/content/en/docs/references/feedback/_index.md]: I don't think HaTS is well known.

File: docs/content/en/docs/references/feedback/_index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Take our [quick 5-question survey](https://forms.gle/BMTbGQXLWSdn7vEs6) to tell us how satisfied you are with
+Skaffold and and what improvements we should make.
+
+## Skaffold Feedback Sessions
+The Skaffold team actively incorporates user feedback as the product evolves.
+We are looking to chat with developers like you to learn about how you use Skaffold and
+anything we can do to make it better suit your needs.
+
+If you’re interested, please provide your contact information below by filling in
```
Comment (balopat) [inline docs/content/en/docs/references/feedback/_index.md]:
```suggestion
If you’re interested, please provide your contact information below by filling out the
```

File: docs/content/en/docs/references/feedback/_index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+[Skaffold feedback form](https://forms.gle/J5h567ncypY2ziq49).
+
+We'll reach out to schedule a one-hour hangouts session at a time convenient for you. Providing your information does not guarantee that we will contact you.
```
Comment (balopat) [inline docs/content/en/docs/references/feedback/_index.md]:
```suggestion

** What is a feedback session? ** 

If you sign up for a feedback session, we'll reach out to schedule a one-hour video conference session at a time convenient for you. No preparation is needed on your part. 
```

File: docs/content/en/docs/references/feedback/_index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+anything we can do to make it better suit your needs.
+
+If you’re interested, please provide your contact information below by filling in
+[Skaffold feedback form](https://forms.gle/J5h567ncypY2ziq49).
+
+We'll reach out to schedule a one-hour hangouts session at a time convenient for you. Providing your information does not guarantee that we will contact you. 
+
+
+No preparation is needed on your part.
+Plus, you'll get the chance to help build something you'll love and a token of our appreciation once participation  is complete!
```
Comment (balopat) [inline docs/content/en/docs/references/feedback/_index.md]: I'm not sure we need this at all. Let's discuss!

File: docs/content/en/docs/references/feedback/_index.md (hunk 3 above)
Comment (balopat) [inline docs/content/en/docs/references/feedback/_index.md]: I would add "Note: providing your information does not guarantee that we will contact you." at the end here somewhere.

File: docs/content/en/docs/references/feedback/_index.md (hunk 7 above)
Comment (tejal29) [reply docs/content/en/docs/references/feedback/_index.md]: this was from the form Sarah had. i can remove it.
